indico/indico-plugins-cern
# Add CloudConvert support

Adds experimental CloudConvert support to the conversion plugin.
Feedback welcome!

### How to test this:
- Make an account on CloudConvert and create an API key (you can also use the sandbox API which doesn't count towards the limit: https://cloudconvert.com/api/v2#overview, but then you can only upload limited files based on an md5 hash)
- Enable CloudConvert instead of Doconverter and set the API key in the Conversion plugin setttings.
- Make sure the webhook (`/conversion/cloudconvert/finished`) is reachable by cloudconvert (use ngrok for reverse proxy if needed)
- Upload a file with a supported extension (e.g. `docx`) and toggle `Convert to PDF`
- Wait & Pray
